### DISCUSSION TITLE
Unable to save config

### DISCUSSION THREAD

**Discussion by YankoValski** [homeassistant-apps/app-cloudflared#289] [Q&A] [answered]
Hi there,

This is great add-on. It works great on my setup. Thanks!

Just one issue I found is that I added additional hosts which is also working.
Now I'm trying to remove the additional config part but it is giving me errors and I'm unable to save it.
What I'm doing wrong here?

Error I'm getting:
`Failed to save add-on configuration, Missing option 'additional_hosts' in root in Cloudflared (abc_cloudflared). Got {'external_hostname': 'mydomain.com', 'tunnel_name': 'homeassistant', 'tunnel_token': ''}`

Thanks

**Comment by brenner-tobias** [comment, marked as answer]
Hi @YankoValski,

the config option itself always needs to be present (unfortunately, this cannot be done differently in HA, since it is an array).
So in order to remove the hosts, you have to add an empty array:

`additional_hosts: []`

Please let me know if this helps.

**Reply by YankoValski** [reply]
Hi Tobias,

Yes, that works.
Thank you!
